.changeset/eight-owls-shop.md

```diff
@@ -0,0 +1,13 @@
+---
+"@comet/admin-icons": minor
+"@comet/cms-admin": minor
+"@comet/admin": minor
+---
+
+Add UI for Impersonation Feature
+
+-   Add indicator to display when impersonation mode is active in `UserHeaderItem`
+-   Add button to allow users to switch on impersonation in the `UserGrid`
+-   Integrate `CrudMoreActionsMenu` in `UserPageToolbar` with an impersonation entry for easy access to this feature.
+-   Add util functions `startPersonation` and `stopPersonation`
+-   Add `impersonate-user` icon
```

Comment:
```suggestion
-   Add `ImpersonateUser` icon
```

This casing will be used for the React component

Reply:
888dbef096c4639f52b8d2c0820c3c611329fac0